test/unit/helpers/xhr/MFA_REQUIRED_multipleU2FmultipleFactors.js

```diff
@@ -0,0 +1,114 @@
+define({
```

Comment:
do we have a `MFA_REQUIRED_multipleU2F.js` already? or can we name this file just that?

Reply:
We have that file. It's for setup with only multiple u2f

Reply:
I think i will rename this file to multiple factors, so we can re-use it for the different factors that support cardinality

Reply:
oh, so the idea is that we can add more factors here that support cardinality? I like that, so we don't have to add one file for each factor